socorro/unittest/processor/rules/test_mozilla.py

```diff
@@ -534,50 +508,26 @@ def test_bad_timestamp_and_no_crash_time(self):
         rule = DatesAndTimesRule()
         rule.act(raw_crash, raw_dumps, processed_crash, processor_meta)
 
-        expected = datetime_from_isodate_string(raw_crash["submitted_timestamp"])
+        expected = datetime.datetime.fromisoformat(raw_crash["submitted_timestamp"])
+        expected_timestamp = int(expected.timestamp())
         assert processed_crash["submitted_timestamp"] == expected
         assert (
             processed_crash["date_processed"] == processed_crash["submitted_timestamp"]
         )
-        assert processed_crash["crash_time"] == 0
-        expected = datetime_from_isodate_string("1970-01-01 00:00:00+00:00")
-        assert processed_crash["client_crash_date"] == expected
-        assert processed_crash["install_age"] == -1335439892
-        assert processed_crash["uptime"] == 0
+        assert processed_crash["crash_time"] == expected_timestamp
+        assert processed_crash["client_crash_date"] == (
+            datetime.datetime.fromtimestamp(expected_timestamp, UTC)
+        )
+        assert processed_crash["install_age"] == 1079701
+        assert processed_crash["uptime"] == 20155
```

Comment:
"uptime" changes here because originally it was based on things based on "timestamp" and we swapped "timestamp" out for "submitted_timestamp".